models/dbt_fullstory/anonymous_users.sql

```diff
@@ -12,12 +29,12 @@ select
   devices.geo_lat_long as last_geo_lat_long,
   devices.event_time as last_event_time,
   devices.event_id as last_event_id
-from {{ ref('devices') }} devices
+from devices
+left join identifies
+  on devices.id = identifies.device_id
 where
   devices.id is not null
+  and identifies.device_id is null
```

Comment:
I may be misunderstanding something, but `device_id` shouldn't ever be null in the raw data set/ Is this intended to be: `identifies.user_id`?

Per https://developer.fullstory.com/destinations/events/#identify

<img width="983" alt="image" src="https://github.com/user-attachments/assets/5ddf2ca5-d88c-412a-b7ca-8509312f6a9b" />